misc/config_tools/board_inspector/cli.py

```diff
@@ -11,11 +11,24 @@
 import lxml.etree
 import argparse
 from importlib import import_module
+from cpuparser import parse_cpuid, get_online_cpu_ids
 
 script_dir = os.path.dirname(os.path.realpath(__file__))
 sys.path.append(os.path.join(script_dir))
 
+def native_check():
+    cpu_ids = get_online_cpu_ids()
+    cpu_id = cpu_ids.pop(0)
+    leaf_1 = parse_cpuid(1, 0, cpu_id)
+    if leaf_1.hypervisor != 0:
+        logging.error(f"Please run this tools in a native OS environment!")
```

Comment:
We avoid saying "please".  Also you might include saying it looks like you're running the tool in a VM.  Perhaps:
"The board inspector must be run in a native OS environment and not from a VM."